l2lchannel.js

```diff
@@ -0,0 +1,128 @@
+import L2LClient from "lively.2lively/client.js";
+import { string, num, promise, fun } from "lively.lang";
+
+export class Channel {
```

Comment:
Is there a reason to copy the channel implementation from channel.js? Does subclassing work? Or just configuring the original channel with a variable backend? (l2l vs local)